docs/development/LedStrip.md

```
@@ -144,35 +112,153 @@ The datasheet can be found here: http://www.adafruit.com/datasheets/WS2812.pdf
+#### Advanced LED configuration
+
+The configuration values for each LED can be displayed in the CLI using the `led` command.
+
+The `led` command with no arguments prints out the current LED configuration, which can be copied for future reference.
+
+Otherwise, `led` expects two arguments - a zero-based LED index number, a space, and then a sequence of parameters in the form:
+` index ,y:ddd:mmm:cc`, where:
+
+an `index` value of 0 refers to the first LED in the strip, 14 to the 1th LED, etc
+`x` and `y` are grid coordinates of a 0 based 16x16 grid,
+`ddd ` is the direction that the LED is pointing in
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)